frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/components/inboxTable.js

```
@@ -0,0 +1,259 @@
+                        onClick={handlePrevious}
+                        disabled={offset === 0}
+                        style={{
+                            padding: "8px 16px",
+                            borderRadius: "8px",
+                            border: "1px solid #ccc",
+                            backgroundColor: offset === 0 ? "#f0f0f0" : "#6B133F",
+                            color: offset === 0 ? "#999" : "#fff",
+                            cursor: offset === 0 ? "not-allowed" : "pointer",
+                            transition: "background-color 0.2s ease"
+                        }}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)